RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -6,5 +6,5 @@
 
 import Foundation
 
-print("Hello, World!")
-
+let game = RockPaperScissorsGame()
```

Comment:
```suggestion
let game: Runnable = RockPaperScissorsGame()
```

위와 같이 변경한다면, 조금더 프로토콜을 사용한 의미가 생길 것 같아요.

Reply:
아..!감사합니다